src/jdk.jstatd/share/man/jstatd.1

```diff
@@ -120,15 +120,12 @@ local security policies should be considered before you start the
 \f[CB]jstatd\f[R] process, particularly in production environments or on
 networks that aren\[aq]t secure.
 .PP
-The \f[CB]jstatd\f[R] server installs an instance of
-\f[CB]RMISecurityPolicy\f[R] when no other security manager is installed,
-and therefore, requires a security policy file to be specified.
-The policy file must conform to Default Policy Implementation and Policy
-File Syntax.
+For security purposes, the jstatd server uses an RMI ObjectInputFilter
+to allow only essential classes to be deserialized.
```

Comment:
It looks like some formatting is now missing around `jstatd`.

Reply:
Right, missed the quotes we use around jstatd in other places.  Added them in this update - I now see the word **jstatd** in bold in the man output like the other places.